### PR TITLE
CLDR-18149 v46.1: Pourover of MessageFormat 2.0 Final Candidate

### PR DESCRIPTION
**Pull request by aphillips**
CLDR-18149

- [x] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Review by macchiati — APPROVED** [review]

**Comment by macchiati** [comment]
Updated the description to add CLDR-18149 at the top.

Is this now final, Addison?

**Comment by macchiati** [comment]
I'm planning on merging this at 13:00 unless I hear otherwise.

**Comment by aphillips** [comment]
Final now.

File: docs/ldml/tr35-messageFormat.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Unicode Locale Data Markup Language (LDML)<br/>Part 9: Message Format
 
-|Version|47 (draft)              |
+|Version|46.1 (draft)            |
```
Comment (pedberg-icu) [inline docs/ldml/tr35-messageFormat.md:5]: This was unnecessary and actually counterproductive. The version on the maint/maint-46 branch, the one targeted for 46.1, already says "46.1 (draft)", so this change may cause a conflict there when cherry-picked.  This version on main is for 47, so we will have to revert this change when we update the spec on main again.

**Comment by pedberg-icu** [comment]
> We'll have to remember to change the version to 47 again after 46.1 is released.

Yes. The version on main should not have had its version changed, the version is already correct on the maint/maint-46 branch.

**Comment by macchiati** [comment]
Peter, should I merge or not?

**Comment by pedberg-icu** [comment]
> Peter, should I merge or not?

@macchiati Yes, please go ahead and merge, for 46.1 any issue can be fixed in conflict resolution when cherry-picking, and for 47 we have time to fix.

**Comment by macchiati** [comment]
done
